tensornetwork/matrixproductstates/mps.py

```diff
@@ -0,0 +1,636 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import numpy as np
+import tensornetwork as tn
```

Comment:
Don't import `tensornetwork` directly. It can lead to circular import of we decided to make `mps` easier to access by putting it in `tensornetwork/__init__.py` file. Prefer to import everything you need directly.